src/main/scala/it/unibo/authsim/library/dsl/cryptography/algorithm/symmetric/DES.scala

```diff
@@ -0,0 +1,48 @@
+package it.unibo.authsim.library.dsl.cryptography.algorithm.symmetric
+
+import it.unibo.authsim.library.dsl.cryptography.algorithm.SymmetricEncryptionAlgorithm
+
+
+trait DES extends SymmetricEncryptionAlgorithm
+
+object DES:
+  import it.unibo.authsim.library.dsl.cryptography.encrypter.asymmetric.key.KeysGenerator
+  import it.unibo.authsim.library.dsl.cryptography.util.ImplicitConversion._
+
+  def apply()= new BasicDES()
+
+  case class BasicDES() extends DES:
```

Comment:
If the class is just used to provide a basic implementation of the trait, it should be made private:
```suggestion
  private case class BasicDES() extends DES:
```